app/views/posts/index.html.erb

```diff
@@ -21,7 +21,7 @@
         <td><%= post.contents %></td>
         <td><%= post.user.username %></td>
         <td><%= link_to 'Show', post %></td>
-        <td><%= link_to 'Destroy', post, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+        <td><%= link_to 'Destroy', post, method: :delete, data: { confirm: 'Are you sure?' }, disabled: :true %></td>
```

Comment:
oops, forgot I but that there, doesn't work nor does it bug out. Will remove when we go to the haml file